RegularApi.Tests/Controllers/Configuration/ApplicationsControllerIT.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using RegularApi.Domain.Views;
+using RegularApi.Tests.Fixtures;
+
+namespace RegularApi.Tests.Controllers.Configuration
+{
+    public class ApplicationsControllerIT : BaseControllerIT
+    {
+        private const string ApplicationUri = "/configuration/applications";
```

Comment:
I don't know if is a good pattern, but with all in uppercase is easily to know that is a constant

Reply:
Well Resharper suggested to name the constant as it is, I can change it, but that means that we'll define our custom C# coding standard and if that is ok, then I'll suggest to drop `_` prefix from `private` attributes